modules/gathering-data-specific-features.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+// * cnv/cnv_users_guide/cnv-collecting-cnv-data.adoc
+// Dependency: modules/cnv-document-attributes.adoc
+
+[id="gathering-data-specific-features_{context}"]
+= Gathering data about specific features
+
+You can gather debugging information about specific features by using the
+`oc adm must-gather` CLI command with the `--image` argument. The `must-gather`
+tool supports multiple images, so you can gather data about more than one
+feature by running a single command.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* The {product-title} Command-line Interface (CLI), commonly known as `oc`.
+
+.Procedure
+
+. Navigate to the directory where you want to store the `must-gather` data.
+
+. Run the `oc adm must-gather` command with the `--image` argument.
+* Run the command with one image:
++
+----
+$ oc adm must-gather --image=<path/to/image>
+----
+* Or, run the command with multiple images. For example:
```

Comment:
I would be more specific in this intro sentence that this command is gathering the default information plus CNV-specific data. (as opposed to just having them have to see it in the callout below).

You could probably even get rid of the first example of a single one, and just do this one.

"For example, the following command gathers both the default cluster data and CNV-specific data:" or something.